unittests/test_utility.py

```diff
@@ -1171,3 +1171,42 @@ def test_cray_cdt_version_no_such_file(tmp_path, monkeypatch):
     rcfile = tmp_path / 'rcfile'
     monkeypatch.setenv('MODULERCFILE', str(rcfile))
     assert os_ext.cray_cdt_version() is None
+
+
+def test_cray_cle_info(tmp_path):
+    # Mock up a CLE release
+    cle_info_file = tmp_path / 'cle-release'
+    with open(cle_info_file, 'w') as fp:
+        fp.write('RELEASE=7.0.UP01\n'
+                 'BUILD=7.0.1227\n'
+                 'DATE=20200326\n'
+                 'ARCH=noarch\n'
+                 'NETWORK=ari\n'
+                 'PATCHSET=09-202003261814\n')
```

Comment:
I could possibly strip off the date from the patchset so as to allow comparisons such as `cray_cle_info().patchset >= '09'`. Now this would compare false.